src/ui/dashboardpage.h

```diff
@@ -30,9 +30,16 @@ class DashboardPage {
         Layout
     };
     void draw(App *app, Dashboard *Dashboard, Mode mode = Mode::View) noexcept;
+    void drawPlots(App *app, DigitizerUi::DashboardPage::Mode mode, Dashboard *dashboard);
+    void drawGrid(float w, float h);
     void drawLegend(App *app, Dashboard *dashboard, const Mode &mode) noexcept;
+
+private:
+    static bool plotButton(App *app, std::string_view glyph, std::string_view tooltip = {});
+
+private:
+    GridLayout plot_layout;
```

Comment:
We follow a style where the public/private fields are on top of the class definition. This way it's easier to assess the required state of the class and potential side-effects. Also, if the state doesn't need to be protected (e.g. no cross dependencies) the class could be a simply a `struct`.